formats/vanish/notification/schema.json

```diff
@@ -0,0 +1,45 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "publishing_app"
```

Comment:
Hmm, actually this is based on the schema for `gone` which only listed `publishing_app` as a requirement. Thinking about it though, I think it does make sense to include `base_path` in that list.